### ISSUE TITLE
Add theme Adaptable to IOMAD theme list

### ISSUE THREAD

**Issue by bguillia** [iomad/iomad#1053]
I am trying to customize the Iomad theme and need a slider and other features for my build within Moodle I have used the [Adaptable theme ](https://moodle.org/plugins/theme_adaptable)and was wondering what it would take to make that theme work in IOMAD.

**Comment by bguillia** [comment]
I attempted to follow: 

> 

The IomadBoost theme overrides a number of the standard renderers to add the custom features. If your own theme has IomadBoost as its parent then it should "just work". If you inherit from some other theme or are building a theme from scratch then you will need to copy the custom renderer code from the IomadBoost theme.
--
 That did not work even after I went through each php and changed theme_bootstrap to theme_adaptable is there something else I should be doing?
